core/whistle_couch-1.0.0/src/wh_couch.hrl

```diff
@@ -66,8 +66,10 @@
                               ,ready = 'false'
                               ,admin = 'false'
                               ,server = #server{}
+                              ,tag = 'local'
```

Comment:
This should be typed as `'local' | ne_binary()` or `atom()` or something, since I doubt we want to tag connections arbitrarily from a type perspective.